### ISSUE TITLE
aread: Error in load() bad restore file magic number 

### ISSUE THREAD

**Issue by nspyrison** [pbiecek/archivist#340]
Potentially depricated file type or corrupted file.
#> Warning: file 'file448841e82f00' has magic number '<html'
#>   Use of save versions prior to 2 is deprecated
#> Error in load(file = tf, envir = .nameEnv): bad restore file magic number (file may be corrupted) -- no data loaded

``` r
library(archivist)
#> Warning: package 'archivist' was built under R version 4.0.5
#> Welcome to archivist (version: 2.3.5).
titanic_imputed <- archivist::aread("pbiecek/models/27e5c")
#> Warning: file 'file448841e82f00' has magic number '<html'
#>   Use of save versions prior to 2 is deprecated
#> Error in load(file = tf, envir = .nameEnv): bad restore file magic number (file may be corrupted) -- no data loaded
titanic_rf <- archivist::aread("pbiecek/models/4e0fc")
#> Warning: file 'file448874aa79ea' has magic number '<html'
#>   Use of save versions prior to 2 is deprecated
#> Error in load(file = tf, envir = .nameEnv): bad restore file magic number (file may be corrupted) -- no data loaded
henry <- archivist::aread("pbiecek/models/a6538")
#> Warning: file 'file4488154f7d45' has magic number '<html'
#>   Use of save versions prior to 2 is deprecated
#> Error in load(file = tf, envir = .nameEnv): bad restore file magic number (file may be corrupted) -- no data loaded
```

<sup>Created on 2021-04-18 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

**Comment by statkclee** [comment]
+1

**Comment by pbiecek** [comment]
thanks
fixed in https://github.com/pbiecek/archivist/commit/18841c9bd216e792299beb4b8d72f287006aee76
will be on cran soon
in the meantime please use
```
load(url("http://ema.drwhy.ai/models/ceb40.rda"))
```

**Comment by nspyrison** [comment]
Confirmed fix with version: 2.3.6 on CRAN.
